mysite/templates/Includes/LoginBar.ss

```diff
@@ -1,41 +1,55 @@
+                <path d="M169.37,40.86H174V34.94l4.79-1.38v7.3H187v4.19h-8.14V61.63a9,9,0,0,0,.78,4.4,2.83,2.83,0,0,0,2.63,1.29,9.14,9.14,0,0,0,2.63-.33c0.71-.22,1.52-0.52,2.39-0.87l1.08,3.65A18.08,18.08,0,0,1,184.81,71a17,17,0,0,1-4.13.48q-3.65,0-5.15-2.06T174,62.52V45.05h-4.66V40.86Z" style="fill:#fff"/>
+                <path d="M207.59,45.47a10.8,10.8,0,0,0-3.41-.6,5.71,5.71,0,0,0-4.07,1.41,6.09,6.09,0,0,0-1.85,3.56V70.79h-4.79V40.86h3.35l0.9,3.65H198a9.16,9.16,0,0,1,2.72-3.14,6.89,6.89,0,0,1,4-1.11,14.51,14.51,0,0,1,3.89.6Z" style="fill:#fff"/>
+                <path d="M211.7,31.77a3.34,3.34,0,0,1,.93-2.37,3.11,3.11,0,0,1,2.36-1,3.4,3.4,0,0,1,2.45,1,3.19,3.19,0,0,1,1,2.37,2.9,2.9,0,0,1-1,2.27,3.57,3.57,0,0,1-2.45.9,3.25,3.25,0,0,1-2.36-.9A3,3,0,0,1,211.7,31.77Zm1,9.1h4.79V70.79h-4.79V40.86Z" style="fill:#fff"/>
+                <path d="M224,40.86h3.29l0.9,3.59h0.24a9.15,9.15,0,0,1,3.55-3.2,10.9,10.9,0,0,1,5-1.11q6,0,8.92,3.53t2.93,11.55a21.8,21.8,0,0,1-1,6.79A14.85,14.85,0,0,1,245,67.14a12.19,12.19,0,0,1-4.34,3.23A13.69,13.69,0,0,1,235,71.5a17.5,17.5,0,0,1-3.5-.27,13.85,13.85,0,0,1-2.78-.93V82.75H224V40.86Zm12.09,3.47a7,7,0,0,0-4.82,1.59,8.85,8.85,0,0,0-2.48,4.81V65.58a8,8,0,0,0,2.46,1.29,12.06,12.06,0,0,0,3.65.45,7.88,7.88,0,0,0,6.55-3.09q2.43-3.08,2.43-9.06a21.94,21.94,0,0,0-.42-4.46,9.82,9.82,0,0,0-1.35-3.41,6.45,6.45,0,0,0-2.39-2.18A7.61,7.61,0,0,0,236.06,44.33Z" style="fill:#fff"/>
+                <path d="M276.18,68.39a13.58,13.58,0,0,1-4.52,2.28,19.56,19.56,0,0,1-5.78.84A14.65,14.65,0,0,1,260,70.4a10.58,10.58,0,0,1-4.13-3.17,13.88,13.88,0,0,1-2.39-4.94,24.91,24.91,0,0,1-.77-6.46q0-7.66,3.41-11.67t9.75-4a18.85,18.85,0,0,1,4.07.45,8.59,8.59,0,0,1,6.29,5.27,15.17,15.17,0,0,1,1,6.07,30,30,0,0,1-.36,4.37H257.63a18.21,18.21,0,0,0,.54,4.67,9.21,9.21,0,0,0,1.67,3.44,7.34,7.34,0,0,0,3,2.15,11.74,11.74,0,0,0,4.46.75,12.13,12.13,0,0,0,4.07-.72,8.64,8.64,0,0,0,3.05-1.68Zm-10.3-24.06a8.29,8.29,0,0,0-5.65,1.86q-2.12,1.85-2.54,6.34h14.84q0-4.55-1.74-6.37A6.46,6.46,0,0,0,265.88,44.33Z" style="fill:#fff"/></svg>
+        </a>
+        <div class="login-form">
+            <h3>Try it yourself</h3>
+            <form action="/Security/LoginForm" method="post" enctype="application/x-www-form-urlencoded" class="form-holder">
```

Comment:
`form-holder` is pretty generic, and you're changing existing styles. I think it makes sense to have a `login-form-holder` class, given this is a special case form for login.